### ISSUE TITLE
gitbook editor is still run loading.Does not display the work area.

### ISSUE THREAD

**Issue by tw-edu-csu-90618137** [GitbookIO/feedback#558]
## Category
the Editor is still run loading 

## What is the current behavior?
I  download the last version editor,login my account,then add new book.
When I want to edit book ,the editor is still run loading, it does not display the work area. 

![Imgur](https://i.imgur.com/QOcuo3y.png "still run loading")

## What is the desired behavior?

the editor run loading a while, then display the work area. 

## Additional information
- Operating System : Windows 7 x64
- Browser version : Chrome 61.0.3163.100 (64 bit)
- Version of the Editor : 7.0.1.2

**Comment by Lin-Buo-Ren** [comment]
Probably related
https://github.com/GitbookIO/gitbook/issues/1873

**Comment by Lin-Buo-Ren** [comment]
I found a workaround to avoid the "loading error" issue, the JSON error appeared to be a Unicode issue in the GitBook website's book info, if you change the "Title" field in the book's setting page to something without non-ASCII and press "Save" botton at the left panel the book will be loadable in local GitBook editor.

**Comment by fengerzh** [comment]
Yes, verified. If `Title` contains Chinese character, it won't open.

**Comment by zonghaishang** [comment]
I have changed tile and description without non-ASCII, but cant't open bitbook.

**Comment by Riseley** [comment]
Almost same, but mine doesn't even load any screen.
